examples/kafka_pubsub/README.md

```diff
@@ -102,4 +90,26 @@ Data source configuration
     "client_id": "tyk-kafka-integration-{{.arguments.name}}"
   }
 }
```

Comment:
The important part is also in lines 442-449 of the json file. It should be added here.

Reply:
I mentioned `field_configs` in the docs.